src/python/ensembl/io/genomio/genome_metadata/compare_genome_stats.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Compare stats in a JSON from NCBI dataset and a json from our core.
+Returns the json from our core including a section with comparisons.
+"""
+
+import json
+from pathlib import Path
+import re
+from typing import Any, Dict
+
+import argschema
+
+
+def _diff_dicts(ncbi: Dict[str, int], core: Dict[str, int]) -> Dict[str, Any]:
+    """Compare two dicts with the same keys and compute the difference of their values.
+
+    Returns:
+        A dict with the same keys, where the values are dicts with 3 keys: ncbi, core, and diff
```

Comment:
```suggestion
        A dict with the same keys, where the values are dicts with 3 keys: ncbi, core, and diff
```
This is not correct, since the returned dictionary only has two keys: `same` and `different`.